app/models/album.rb

```diff
@@ -4,7 +4,7 @@ class Album < ActiveRecord::Base
 
   attr_accessor(:image_upload, :photographer_user,
                 :photographer_name)
-
+  default_scope { order(start_date: :desc) }
```

Comment:
default_scope considered harmful. http://stackoverflow.com/questions/25087336/why-is-using-the-rails-default-scope-often-recommend-against